lib/active_record_shards/sql_comments.rb

```diff
@@ -1,10 +1,10 @@
-# show which connection was picked to debug master/replica slowness when both servers are the same
+# show which connection was picked to debug primary/replica slowness when both servers are the same
 module ActiveRecordShards
   module SqlComments
     module Methods
       def execute(query, name = nil)
         replica = ActiveRecord::Base.current_shard_selection.on_replica?
-        query += " /* #{replica ? 'replica' : 'master'} */"
+        query += " /* #{replica ? 'replica' : 'primary'} */"
```

Comment:
This might deserve a mention in the changelog. Clients may have been using those comments to perform operations.